pkg/csi_driver/node.go

```diff
@@ -49,15 +52,29 @@ type nodeServer struct {
 	mounter     mount.Interface
 	metaService metadata.Service
 	volumeLocks *util.VolumeLocks
+	kubeClient  kubernetes.Interface
 }
 
-func newNodeServer(driver *GCFSDriver, mounter mount.Interface, metaService metadata.Service) csi.NodeServer {
-	return &nodeServer{
+func newNodeServer(driver *GCFSDriver, mounter mount.Interface, metaService metadata.Service, featureLockRelease bool) (csi.NodeServer, error) {
```

Comment:
we can use options for any future features too

Reply:
Done